### DISCUSSION TITLE
Update documentation for "Using a Custom Font with Zettlr"

### DISCUSSION THREAD

**Discussion by dan-todd** [Zettlr/Zettlr#4985] [General]
The example on this page seems to be incorrect as of v3.0.5. 

.cm-editor should be changed to .cm-content

I believe the new CSS should be:
```
body .main-editor-wrapper .cm-content{
     font-family: '<your-font-name here>', <placeholder>;
}
```

**Comment by lordofscripts** [comment]
I want to change the editor font, where do I do that in the latest version of Zettlr? @dan-todd

**Comment by lordofscripts** [comment]
Just added that to the Custom CSS with my fonts and it worked! thanks a million @dan-todd  !
